src/parser.ts

```diff
@@ -433,10 +434,9 @@ via the \`definition\` "${key}".`
 
 type Definitions = {[k: string]: LinkedJSONSchema}
 
-/**
- * TODO: Memoize
- */
-function getDefinitions(
+const getDefinitions = memoize(getDefinitionsInner)
+
+function getDefinitionsInner(
```

Comment:
Two small nits:

1. Would you mind using Lodash's `memoize` function, to avoid introducing the new dependency on `memoizee`?
2. A stylistic nit: could you avoid `getDefinitionsInner`, and instead inline the function? `Inner`, `Wrapper`, `Impl`, etc. often feel smelly to me. (Alternatively, call it `getDefinitions` and `getDefinitionsMemoized` to make the naming more meaningful.)

Reply:
Done! I've not inlined the function as it needs to call itself, but have improved the name and switched to using `lodash` :)